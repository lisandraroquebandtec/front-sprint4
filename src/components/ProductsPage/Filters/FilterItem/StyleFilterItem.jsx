import styled from "styled-components";

export const FiltersItemContainer = styled.li`
  display: flex;
  align-items: center;
  padding: 12px 14px;
  :hover {
    cursor: pointer;
    background-color: rgba(38, 30, 30, 0.04);
  }
`;

export const FiltersLabel = styled.span`
  text-transform: uppercase;
  font-size: 0.6875rem;
  font-weight: 400;
  color: #261e1e;
`;

export const FilterImg = styled.img`
  height: 20px;
  margin-left: 8px;
`;
