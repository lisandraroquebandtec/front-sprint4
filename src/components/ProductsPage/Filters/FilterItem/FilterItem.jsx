import { FiltersItemContainer,FiltersLabel, FilterImg} from "./StyleFilterItem";
import filter from "../../../../assets/images/filter.svg";
function FilterItem(props) {
  return (
    <FiltersItemContainer>
      <FiltersLabel>{props.filters}</FiltersLabel>
      <FilterImg src={filter}/>
    </FiltersItemContainer>
  );
} 
export default FilterItem;

