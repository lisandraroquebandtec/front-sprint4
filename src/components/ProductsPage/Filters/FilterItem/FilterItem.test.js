import React from 'react';
import { render} from '@testing-library/react';
import FilterItem from './FilterItem';

describe('O componente de item do Filter,', () =>{
  it('o snapshot do componente deve permanecer sempre o mesmo', ()=>{
    const {container} = render(
    <FilterItem
    img= '/assets/images/filter.svg'
    />)
    expect(container.firstChild).toMatchSnapshot();
  })
})