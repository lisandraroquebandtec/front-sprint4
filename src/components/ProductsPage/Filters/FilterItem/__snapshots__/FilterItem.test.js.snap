// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`O componente de item do Filter, o snapshot do componente deve permanecer sempre o mesmo 1`] = `
<li
  class="sc-bdnxRM hIFmCO"
>
  <span
    class="sc-gtsrHT bAmSmd"
  />
  <img
    class="sc-dlnjwi ckhPXh"
    src="filter.svg"
  />
</li>
`;

exports[`O componente de item do produto, o snapshot do componente deve permanecer sempre o mesmo 1`] = `
<li
  class="sc-bdnxRM hIFmCO"
>
  <span
    class="sc-gtsrHT bAmSmd"
  />
  <img
    class="sc-dlnjwi ckhPXh"
    src="filter.svg"
  />
</li>
`;
